ocrd_network/ocrd_network/utils.py

```diff
@@ -147,3 +148,20 @@ def stop_mets_server(mets_server_url: str) -> bool:
     if response.status_code == 200:
         return True
     return False
+
+
+def validate_workflow(workflow: str, logger=None) -> bool:
+    """ Check that workflow is not empty and parseable to a lists of ProcessorTask
+    """
+    if workflow.strip() == "":
+        if logger:
+            logger.info("Workflow is invalid (empty string)")
+        return
+    try:
+        tasks_list = workflow.splitlines()
+        [ProcessorTask.parse(task_str) for task_str in tasks_list if task_str.strip()]
+    except BaseException as e:
```

Comment:
```suggestion
    except Exception as e:
```

But even better, let `ProcessorTask.parse` raise (more specific) `ValueError`.

Reply:
I have implemented your suggestion to [raise ValueError](https://github.com/OCR-D/core/pull/1143/commits/d695f4955dfbc7d4be166ebb2db9520a1d7b7524)